selfdrive/car/honda/interface.py

```diff
@@ -75,6 +75,7 @@ def _get_params(ret, candidate, fingerprint, car_fw, experimental_long, docs):
       ret.longitudinalActuatorDelayUpperBound = 0.5 # s
       if candidate in HONDA_BOSCH_RADARLESS:
         ret.stopAccel = CarControllerParams.BOSCH_ACCEL_MIN  # stock uses -4.0 m/s^2 once stopped but limited by safety model
+        ret.stoppingDecelRate = 0.3  # reach stopping target smoothly
```

Comment:
can also tune vEgoStopping, perhaps it's a bit too high at ~1 mph. can you look at some stock routes and see what it does coming to a stop? at what speed does it really start to ramp down to -4 m/s^2?

Reply:
I was going to set it to 0.25 as with gm/toyota but according to these screenshots it looks like 0.3 is a more accurate speed.

<img width="1624" alt="Screenshot 2023-08-05 at 10 25 11 AM" src="https://github.com/commaai/openpilot/assets/1976269/7bc77801-b153-4365-b39b-186d28b362a2">
<img width="1624" alt="Screenshot 2023-08-05 at 10 26 55 AM" src="https://github.com/commaai/openpilot/assets/1976269/24100c16-4e40-4b9d-b80c-3b122ee6f9ae">
<img width="1624" alt="Screenshot 2023-08-05 at 10 28 15 AM" src="https://github.com/commaai/openpilot/assets/1976269/a6f1a18c-8ecc-4529-a7e7-0becf8a331aa">
<img width="1624" alt="Screenshot 2023-08-05 at 10 29 04 AM" src="https://github.com/commaai/openpilot/assets/1976269/a02b5706-596e-46a8-b5ec-5f93558a603c">


```suggestion
        ret.vEgoStopping = 0.3
```



deceleration minimums around 2.5.

<img width="1624" alt="Screenshot 2023-08-05 at 10 27 25 AM" src="https://github.com/commaai/openpilot/assets/1976269/6ecab7e4-4e9b-483d-b7e4-f8f1cf4a73b2">
<img width="1624" alt="Screenshot 2023-08-05 at 10 31 28 AM" src="https://github.com/commaai/openpilot/assets/1976269/86f710c0-3cca-4dca-bfc6-b7ad53d6a3bb">
<img width="1624" alt="Screenshot 2023-08-05 at 10 32 28 AM" src="https://github.com/commaai/openpilot/assets/1976269/de1dd520-9748-4bd1-b72b-fd0f4f495d60">

`5130484aa8069bad|2023-08-03--16-23-11`

Reply:
<img width="1197" alt="Screenshot 2023-08-07 at 2 45 43 PM" src="https://github.com/commaai/openpilot/assets/1976269/e383855b-aee9-4f49-80dc-32b5db86769f">

`1a5d045d2c531a6d|2023-08-07--13-59-44`

wasn't any better, so currently using `ret.stoppingDecelRate = 0.15` with decent results tested on both downhills and uphills.

Reply:
so the stock system sets it to 0.5 well before stopping, then transitions to -4.0 immediately:

![image](https://github.com/commaai/openpilot/assets/25857203/99c9ff65-e966-4d3f-b899-9ae242fe3eb5)